cli/assets/bundle/html-page.hbs

```diff
@@ -0,0 +1,44 @@
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no"
+    />
+    {{#if html.description}}
+    <meta name="description" content="{{html.description}}" />
+    {{/if}}
+    {{#if html.iconUrl}}
+    <link rel="shortcut icon" href="{{html.iconUrl}}" />
+    {{/if}}
+    <title>{{html.title}}</title>
+    <style type="text/css" id="wasm4-css">{{{html.wasm4Css}}}</style>
+  </head>
+  <body>
+    <div class="container">
+      <div id="content">
+        <div class="infobox">
+          <h2 id="title"></h2>
+          <h4 id="author"></h4>
+          <span class="play-button"></span>
+        </div>
+        <div id="screenshot"></div>
+      </div>
+    </div>
+    <div class="gamepad">
+      <div id="gamepad-dpad"></div>
+      <div id="gamepad-action1"></div>
+      <div id="gamepad-action2"></div>
+    </div>
+    <script
+      id="wasm4-cart-json"
+      type="application/json"
+    >{{{html.wasmCartJson}}}</script>
+    <div id="wasm4-bundle-info">
+    {{#each buildInfo}}
+      <input type="hidden" name="{{this.name}}" value="{{this.value}}">
```

Comment:
Would it be better for SEO to use `<meta>` tags?

```html
<meta name="generator" content="WASM-4 1.0.3">
<meta name="created" content="2021-09-07T23:04:21.553Z">
```

Reply:
Sure,
I've always added commit-id, version & build-timestamp inside rendered html  pages for debugging reasons,
but I guess it this context makes more sense to use the proper meta tags.